CHANGES.rst

```diff
@@ -7,6 +7,7 @@ Version 3.4.0 (2019-09-XX)
 ==========================
 - Add support for pyarrow 0.14.1
 - Use urlquote for faster quoting/unquoting
+- ``MetaPartition.load_dataframes`` now raises if table in ``columns`` argument doesn't exist
```

Comment:
3.4.0 was released already. Can you please move this to a new section?

Reply:
Added a new section to the changelog